README.md

```diff
@@ -1,6 +1,25 @@
+![LOGO](https://user-images.githubusercontent.com/62508697/139729943-d46fc1ee-9344-4563-9d7d-2369f38fce54.jpeg)
```

Comment:
very nice!!!
maybe it will be better to have this jpeg as part of this repo
IMHO is better to have fewer dependencies in other resources